contracts/Treasury.sol

```diff
@@ -88,20 +86,16 @@ contract Treasury is Context, ReentrancyGuard {
     ////////////////////////////// Only Governor //////////////////////////////
     /**
      * @notice Add token into treasury management system
-     * @dev Add token address in whitelistedTokens list and add cToken in mapping
+     * @dev Add token address in whitelistedTokens list and add comet in mapping
      * @param _token address which we want to add in token list.
-     * @param _cToken CToken address correspond to _token
+     * @param _comet comet address correspond to _token
      * @param _oracle Chainlink oracle address for token/USD feed
      */
-    function addWhitelistedToken(
-        address _token,
-        address _cToken,
-        address _oracle
-    ) external onlyGovernor {
+    function addWhitelistedToken(address _token, address _comet, address _oracle) external onlyGovernor {
         require(_token != address(0), "token-address-is-zero");
-        require(_cToken != address(0), "cToken-address-is-zero");
+        require(_comet != address(0), "comet-address-is-zero");
         require(_oracle != address(0), "oracle-address-is-zero");
-        _addToken(_token, _cToken, _oracle);
+        _addToken(_token, _comet, _oracle);
```

Comment:
add validation in _addToken that token and _comet are correct

Reply:
Done